packages/design-system/src/components/Dropdown/Primitive/DropdownTitle.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { forwardRef, Ref } from 'react';
+
+import styles from './DropdownTitle.module.scss';
+
+export type DropdownTitleType = { children: string };
+
+const DropdownTitle = forwardRef(
```

Comment:
Wow, TIL: optgroup in select seem to hide the titles from screenreaders. The list is flatten like a simple select
So this implementation is "totally fine"